lib/tasks/repair_tess.rake

```diff
@@ -0,0 +1,24 @@
+namespace :data_repair do
+  task :tess do
+    query = %q(
+      select user_id
+      from user_reductions
+      where reducible_type='Workflow' and reducible_id=11235
+      group by reducible_type, reducible_id, reducer_key, user_id, subgroup
+      having count(*) > 1
+    )
+
+    affected_items = ActiveRecord::Base::Connection.execute(query)
+    affected_items.each do |user_id|
+      ActiveRecord::Base.transaction do
+        UserReductions.where(
+          reducible_type: 'Workflow',
+          reducible_id: 11235,
+          user_id: user_id,
+        ).delete_all
+
+        FetchClassificationsWorker.perform_async(11235, user_id, FetchClassificationsWorker.fetch_for_user)
```

Comment:
`panoptes.get_user_classifications` will page results for users with more than 20 or so classifications. Will the worker page through them correctly as it processes them?

Reply:
yep, it's the same mechanism that we used to use for backfilling workflows, so it's used to getting lots and lots of classifications